inspirehep/modules/theme/jinja2filters.py

```diff
@@ -33,6 +33,7 @@
 
 from flask import session, current_app
 from jinja2.filters import evalcontextfilter
+from inspirehep.utils.jinja2 import render_template_to_string
```

Comment:
Since you are also touching this file, please add
```python
from __future__ import (
    absolute_import,
    division,
    print_function,
    unicode_literals)
```
here as well.